pinot-core/src/main/java/org/apache/pinot/core/operator/combine/GroupByOrderByCombineOperator.java

```diff
@@ -235,11 +235,15 @@ protected IntermediateResultsBlock mergeResults()
     }
 
     IndexedTable indexedTable = _indexedTable;
-    indexedTable.finish(false);
+    if (indexedTable != null) {
```

Comment:
I saw several extra `null` checks introduced. Want to understand why because most likely this is because something is not wired correctly

Reply:
@Jackie-Jiang This is from a test where all input values are nulls. In this case _table is null. Check `AllNullsQueriesTest`.

Reply:
Not needed anymore.